lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -89,9 +91,27 @@ class FullPageScreenshot extends FRGatherer {
       screenOrientation: {angle: 0, type: 'portraitPrimary'},
     });
 
-    // TODO: elements collected earlier in gathering are likely to have been shifted by now.
-    // The lower in the page, the more likely (footer elements especially).
-    // https://github.com/GoogleChrome/lighthouse/issues/11118
+    // Now that the viewport is taller, give the page some time to fetch new resources that
+    // are now in view.
+    const networkMonitor = new NetworkMonitor(context.driver.defaultSession);
+    const waitForNetworkIdleResult = waitForNetworkIdle(session, networkMonitor, {
+      networkQuietThresholdMs: 1000,
+      busyEvent: 'network-2-busy',
```

Comment:
hmm.. should the waitfor be called before `setDeviceMetricsOverride` (but only awaited later like it is now)?

Reply:
probably

Reply:
the answer is no

`isIdle` callback handles the initial state.